authority/provisioner/jwk.go

```diff
@@ -152,19 +151,15 @@ func (p *JWK) AuthorizeSign(ctx context.Context, token string) ([]SignOption, er
 		claims.SANs = []string{claims.Subject}
 	}
 
-	dnsNames, ips, emails := x509util.SplitSANs(claims.SANs)
-	return []SignOption{
+	return append([]SignOption{
 		// modifiers / withOptions
 		newProvisionerExtensionOption(TypeJWK, p.Name, p.Key.KeyID),
 		profileDefaultDuration(p.claimer.DefaultTLSCertDuration()),
 		// validators
 		commonNameValidator(claims.Subject),
 		defaultPublicKeyValidator{},
-		dnsNamesValidator(dnsNames),
-		emailAddressesValidator(emails),
-		ipAddressesValidator(ips),
 		newValidityValidator(p.claimer.MinTLSCertDuration(), p.claimer.MaxTLSCertDuration()),
-	}, nil
+	}, sansValidators(claims.SANs)), nil
```

Comment:
I don't think this is right, it works because SignOption is an interface, but I think the result array will look like:
```go
[]SignOption{
   newProvisionerExtensionOption(TypeJWK, p.Name, p.Key.KeyID),
   ...
   newValidityValidator(p.claimer.MinTLSCertDuration(), p.claimer.MaxTLSCertDuration()),
   []SignOption {
        dnsNamesValidator(dnsNames),
        ....
   },
}
```

Reply:
`sansValidators(claims.SANs)...` will work, but we might want to convert that sansValidators to a real validator so we don't need to append here and we can just add it in the array.